addons/mail/models/mail_channel.py

```diff
@@ -836,6 +836,7 @@ def channel_info(self, extra_info=False):
                 partner_channel = channel_partners.filtered(lambda pc: pc.partner_id.id == self.env.user.partner_id.id)
                 if partner_channel:
                     partner_channel = partner_channel[0]
+                    info['members'] = partner_channel.mail_channel_partner_format(partner_infos)
```

Comment:
As I posted earlier, this makes it look like it is the full member list but it's not the case. Another key should be used in the result to give the member info of the current partner only.

Reply:
Note: this is actually making `channel_info` wrong to ever broadcast on the channel bus itself, because it contains member specific information.

This is actually a big problem because other partners might receive the info from whichever partner triggered the bus notification in the first place.

The least that can be done is to ensure in the convertData to ignore this key if the partner is doesn't match the current partner id, but ideally this should not happen at all.

Also even if we ignore bad information, it's still a leak of info, and it's still also a problem because we won't have the actual member id for the current partner then.

Reply:
The second comment can be ignored but my first comment is still applicable here. It's not ok to use the `members` key to give information about the current partner only. This makes it look like there is only one member in the channel.